.gitlab-ci.yml

```diff
@@ -0,0 +1,124 @@
+image: golang:1.11-alpine3.9
+
+cache:
+  paths:
+    - /go/src/github.com
+    - /go/src/golang.org
+    - /go/src/google.golang.org
+    - /go/src/gopkg.in
+
+variables:
+   DOCKER_HOST: tcp://docker:2375/
+   DOCKER_DRIVER: overlay2
```

Comment:
What is this for? Do I need it in the Mender QA pipeline?

Reply:
that used to be required by Gitlab for dind to work, it is now the default volume driver so I'll remove